data/crac/crac-impl/src/main/java/com/powsybl/openrao/data/cracimpl/SwitchPairImpl.java

```diff
@@ -57,6 +58,14 @@ public NetworkElement getSwitchToClose() {
         return switchToClose;
     }
 
+    @Override
+    public boolean isCompatibleWith(ElementaryAction otherElementaryAction) {
+        if (otherElementaryAction instanceof SwitchPair switchPair) {
+            return !switchToOpen.equals(switchPair.getSwitchToOpen()) && !switchToOpen.equals(switchPair.getSwitchToClose()) && !switchToClose.equals(switchPair.getSwitchToClose()) && !switchToClose.equals(switchPair.getSwitchToOpen());
```

Comment:
I'm not sure this is OK. We may have a bus bar change that changes bus from 1 to 2, another from 2 to 3. combining both allows the RAO to move from 1 to 3, but this method wouldn't allow it anymore.
what do you think @phiedw ?

Reply:
I'm thinking that "icompatible" RAs should be viewed as one RA that completely overrides the effect of the other, not just modifies part of its effect

Reply:
I think we should ask Sebastien what the TSOs want in this case in particular, the approach here is the most "safe" but perhaps indeed they allow multiple bus bar changes for the same line (but imo they would define the switch from 1 to 3 directly if that were the case)

Reply:
This is currently only used to go from 1 node topo to 2 node topo and other way round, so the case where we would move a line from bus bar 1 to 2, and then to 3 can probably be ignored for now (since it would also require some changes in other parts of the RAO to make sure things go smoothly, eg we currently just apply all network actions selected in some parts of the code and we would need to make sure we applied in the correct order for this case).

So I think for now this version is good.